web/controllers/userController/index.ts

```diff
@@ -47,8 +48,6 @@ const logger = getLogger();
  * curl -k -i -X GET -H "Token: <AUTHTOKEN>" https://api.corona-school.de/api/user
  *
  * @apiUse User
- * @apiUse Subject
- * @apiUse Match
```

Comment:
I saw that Subject and Match are note used in this endpoint. I understanding is that you put only those which are used in the specific endpoint.

Reply:
I think they existed because they are implicitly used by the `User` object, which refers to `Subject` and `Match`, so one needs to know what `Subject` and `Match` are. For example, see the generated docs [here](https://api.docs.corona-school.de/97-mentor-update/#api-User-getCurrentUser).

Reply:
Okay, I thought only directly used objects are listed in the apidocs blocks. Then I will undo it.